src/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      givenTemp: 82,
+      convertToCelsius: true
+    }
+
+    this.toggleConversion = this.toggleConversion.bind(this)
+    this.updateGivenTemp = this.updateGivenTemp.bind(this)
+    this.convert = this.convert.bind(this)
```

Comment:
Sounds like you don't have [vscode setup properly](https://github.com/Microsoft/vscode/issues/2394)  or are [using an old version](https://github.com/Microsoft/vscode/issues/408)